### PR TITLE
Added youtube playlists for React and Flask

### PR DESCRIPTION
**Pull request by Bhu11154**
## What does this PR do?
Add resource(s) | Improve repo

### Why is this valuable (or not)?
Yes, they are really good content for the above frameworks.
### How do we know it's really free?
They are youtube playlists :)
### For book lists, is it a book? For course lists, is it a course? etc.
Youtube playlists. 
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Don't invent titles! Use the title given by the resource.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -651,10 +651,11 @@
 * [React Crash Course 2020](https://www.youtube.com/watch?v=4UZrsTqkcW4) - freecodecamp
 * [React Hooks](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH8EtggFGERCwMY5u5hOjf-h) - WebDevSimplified
 * [ReactJS Basics](https://www.youtube.com/playlist?list=PLe30vg_FG4OSw8SIcLVci-jB_-W1ZkLYp) - Bitfumes
+* [ReactJS Basics to Advanced and 30+ Projects](https://www.youtube.com/playlist?list=PLWKjhJtqVAbkArDMazoARtNz1aMwNWmvC) - FreeCodeCamp
 * [ReactJS Frontend Web Development For Beginners](https://www.udemy.com/course/react-tutorial/) - Ryan Dhungel (Udemy)
 * [ReactJS Full Course for Beginners \| Complete All-in-One Tutorial \| 9 Hours](https://www.youtube.com/watch?v=RVFAyFWO4go) - Dave Gray
 * [ReactJS Tutorial for Beginners](https://www.youtube.com/playlist?list=PLC3y8-rFHvwgg3vaYJgHGnModB54rxOk3) - Codevolution
-* [Start Using React to Build Web Applications](https://egghead.io/courses/react-fundamentals)
+* [Start Using React to Build Web Applications](https://egghead.io/courses/react-fundamentals)
```
Comment (eshellman) [inline courses/free-courses-en.md:658]: lint

Comment (davorpa) [reply courses/free-courses-en.md:658]:
```suggestion
* [Start Using React to Build Web Applications](https://egghead.io/courses/react-fundamentals) - Joe Maddalone
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -913,6 +914,7 @@
 
 * [Flask Fridays](https://www.youtube.com/playlist?list=PLCC34OHNcOtolz2Vd9ZSeSXWc8Bq23yEz) - Codemy.com (YouTube)
 * [Flask Tutorials](https://www.youtube.com/playlist?list=PL-osiE80TeTs4UjLw5MM6OjgkjFeUxCYH) - Corey Schafer
+* [Flask Tutorials for Beginners](https://www.youtube.com/playlist?list=PLzMcBGfZo4-n4vJJybUVV3Un_NFS5EOgX) - Tech with Tim
```
Comment (davorpa) [inline courses/free-courses-en.md:917]: Use titles as provided by original source

```suggestion
* [Flask Tutorials](https://www.youtube.com/playlist?list=PLzMcBGfZo4-n4vJJybUVV3Un_NFS5EOgX) - Tech with Tim
```

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -651,10 +651,11 @@
 * [React Crash Course 2020](https://www.youtube.com/watch?v=4UZrsTqkcW4) - freecodecamp
 * [React Hooks](https://www.youtube.com/playlist?list=PLZlA0Gpn_vH8EtggFGERCwMY5u5hOjf-h) - WebDevSimplified
 * [ReactJS Basics](https://www.youtube.com/playlist?list=PLe30vg_FG4OSw8SIcLVci-jB_-W1ZkLYp) - Bitfumes
+* [ReactJS Basics to Advanced and 30+ Projects](https://www.youtube.com/playlist?list=PLWKjhJtqVAbkArDMazoARtNz1aMwNWmvC) - FreeCodeCamp
```
Comment (davorpa) [inline courses/free-courses-en.md:654]: - Use titles as provided by original source
- Alphabetize

```suggestion
* [React Tutorials](https://www.youtube.com/playlist?list=PLWKjhJtqVAbkArDMazoARtNz1aMwNWmvC) - freeCodeCamp
* [ReactJS Basics](https://www.youtube.com/playlist?list=PLe30vg_FG4OSw8SIcLVci-jB_-W1ZkLYp) - Bitfumes
```

**Review by davorpa — APPROVED** [review]

**Comment by Bhu11154** [comment]
Hi! Are there any changes to be made? Can you merge my pull request? Thanks!

**Comment by davorpa** [comment]
> Hi! Are there any changes to be made? Can you merge my pull request? Thanks!

No. Please, be patient

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**